code/src/layouts/call-to-action/components/call-to-action-item/CallToActionItem.tsx

```diff
@@ -0,0 +1,49 @@
+import AppBox from "@/components/app-box/AppBox";
+import AppTypography from "@/components/app-typography/AppTypography";
+import AppButton from "@/components/app-button/AppButton";
+import AppLink from "@/components/app-link/AppLink";
+
+type CallToActionSectionItemProps = {
+  titleTranslationKey: string;
+  descriptionTranslationKey: string;
+  captionTranslationKey: string;
+  linkTo: string;
+  imageUrl: string;
+  buttonLabelTranslationKey: string;
+};
+
+const CallToActionItem = ({
+  titleTranslationKey,
+  descriptionTranslationKey,
+  captionTranslationKey,
+  buttonLabelTranslationKey,
+  linkTo,
+  imageUrl
+}: CallToActionSectionItemProps) => {
+  return (
+    <AppBox
+      className="call-to-action__item"
+      sx={{ backgroundImage: `url(${imageUrl})` }}
```

Comment:
for now we configured `webpack` with opportunity to write `bg image` inside `scss`
try to rewrite

Reply:
I knew about new `webpack` config, you did a really great job! I thought it would be better if we can store our images inside constants for that folder and not inside scss. What do you think about that?

Reply:
> I know about new `webpack` config, you did a really great job! I thought it would be better if we can store our images inside constants for that folder and not inside scss. What do you think about that?

Not sure. But for now we replaced using inline styles to `scss`. We should discuss it. Or rewrite existing approach with bg image for our `banner` at the home page. And follow one approach as well